<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CamWindow</class>
 <widget class="QWidget" name="CamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CamWindow</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>800</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>1000</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/back.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1060</y>
     <width>801</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="AlbumBtn">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icon/album.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="TakePhoto">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icon/take_photo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="RecordBtn">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="PrePicBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="NextPicBtn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>500</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteBtn">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/garbage.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/play.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
